code/game/objects/items/weapons/weaponry.dm

```diff
@@ -148,16 +148,20 @@
 /obj/item/weapon/katana/magic
 	name = "enchanted sword"
```

Comment:
requesting a name change to "moonlight-enchanted sword"

Reply:
can do

Reply:
![image](https://user-images.githubusercontent.com/17928298/53008039-44857580-3417-11e9-887f-04f5550f2cdf.png)